recipes-debian/sbsigntool/sbsigntool_debian.bb

```diff
@@ -0,0 +1,52 @@
+# base recipe: meta-luv/recipes-devtools/sbsigntool/sbsigntool_git.bb
+# base branch: master
+# base commit: ffb7d193abfb0c11ed5f0133aca039daddbf9c16
+SUMMARY = "Signing utility for UEFI secure boot"
+
+LICENSE = "GPLv3"
+LIC_FILES_CHKSUM = "file://LICENSE.GPLv3;md5=9eef91148a9b14ec7f9df333daebc746"
+
+inherit autotools-brokensep pkgconfig
+inherit debian-package
+require recipes-debian/sources/sbsigntool.inc
+
+SRC_URI += " \
+    file://fix-mixed-implicit-and-normal-rules.patch;apply=0 \
+    file://disable-man-page-creation.patch \
+    file://0001-configure-Fixup-build-dependencies-for-cross-compili.patch \
+    "
+
+BBCLASSEXTEND = "native"
+
+DEPENDS = "binutils openssl gnu-efi util-linux"
+
+do_configure_class-native() {
+    if [ ! -e lib/ccan.git/Makefile ]; then
```

Comment:
According to the debian source package,  debian source doesn't contain .git directory. so this code seems doesn't need it.

```
masami@ubuntu1804:~/tmp/sbsigntool-0.9.2/lib$ pwd
/home/masami/tmp/sbsigntool-0.9.2/lib
masami@ubuntu1804:~/tmp/sbsigntool-0.9.2/lib$ find . -name .git
masami@ubuntu1804:~/tmp/sbsigntool-0.9.2/lib$ 
```